packages/babel-preset-env/test/fixtures/plugins-integration/issue-7527/exec.js

```diff
@@ -0,0 +1,6 @@
+class MyDate extends Date {
+  constructor(time) {
+      super(time);
+  }
+}
+let myDate = new MyDate();
```

Comment:
Can we also add an assertion here? To make sure this is a real date instance.

Something like `assert.equal(typeof myDate.toString(), 'string');` should work.